base-cert-manager/statefulset.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: cockroachdb
+  labels:
+    app: cockroachdb
+  annotations:
+    "app.uw.systems/description": "Used to store data projections of events in multiple services."
+    "app.uw.systems/tier": "tier_1"
+    "app.uw.systems/repos.cockroachdb-manifests": "https://github.com/utilitywarehouse/cockroachdb-manifests"
+    "app.uw.systems/tags.oss": "true"
+    "secret.reloader.stakater.com/reload": "cockroachdb-ca, cockroachdb.client.root, cockroachdb.node"
```

Comment:
I think `cockroachdb.node` is enough here - you don't want to be rolling the stateful set when the client cert changes.